oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticBulkProcessorHandler.java

```diff
@@ -250,6 +259,7 @@ public void afterBulk(long executionId, BulkRequest bulkRequest, BulkResponse bu
         @Override
         public void afterBulk(long executionId, BulkRequest bulkRequest, Throwable throwable) {
             LOG.error("ElasticIndex Update Bulk Failure : Bulk with id {} threw an error", executionId, throwable);
+            ElasticBulkProcessorHandler.this.ioException = new IOException(throwable);
```

Comment:
If the throwable is already an IOException, then wrapping could be avoided. But it's probably not worth the trouble.